genie-web/src/main/resources/genie-web-defaults.yml

```diff
@@ -118,6 +118,9 @@ spring:
     name: genie
   banner:
     location: genie-banner.txt
+  cache:
+    caffeine:
+      spec: maximumSize=100,expireAfterAccess=600s
```

Comment:
Quick back-of-the-envelope calculation specific to our internal deployment/usage.

We peak at 1.2k jobs daily in tests, with a 10 nodes cluster.
If clients are polling for outputs (which they are), the manifest cache may reach 120 'hot' elements.
Depending on the frequency of such requests, this may lead to some trashing in the cache.

i.e. this default value is a bit low for our own deployment.
Just something to keep in mind, we may want to double it when rolling this out.

Reply:
Yeah this is just the default we can change it in internal